### PR TITLE
Change loki chart dependency to stand alone loki

### PR DESCRIPTION
**Pull request by jacobcrawford**
We do not need the loki stack w. promtail and other dependencies in this chart as we are moving to Grafana agents

**Description of your changes:**

Checklist:

* [x] I have bumped the chart version
* [x] I have documented my changes if this is needed
* [x] I have described my changes in the "description of your changes" field
* [x] I have tested this change on a running cluster and the Argocd dashboard shows healthy and synced
* [ ] I have created the necessary tests in the chart, if they are possible/necessary
* [x] I have squashed commits if necessary

### CODE REVIEW

File: charts/loki/chart/values.yaml [hunk 1]
```diff
@@ -1,15 +1,2 @@
...
-  fluent-bit:
-    enabled: false
-
-  grafana:
-    enabled: false
-
-  prometheus:
-    enabled: false
+# https://github.com/grafana/helm-charts/blob/main/charts/loki-distributed/values.yaml
+loki-distributed:
```
Comment (klausenbusk) [inline charts/loki/chart/values.yaml:2]: Shouldn't this be:
```suggestion
loki-distributed: {}
```

Right now you are setting it to a blank string (we had issues with this in the past).

**Comment by jacobcrawford** [comment]
image

**Review by MadsHT — APPROVED** [review]
LGTM

**Review by skovbakke — APPROVED** [review]
LGTM

**Pull request merged**